glide.yaml

```diff
@@ -1,32 +1,10 @@
 package: github.com/eris-ltd/eris-db
 import:
+- package: github.com/tendermint/tendermint
 - package: github.com/gin-gonic/gin
 - package: github.com/gorilla/websocket
 - package: github.com/naoina/toml
-- package: github.com/tendermint/ed25519
-- package: github.com/tendermint/go-common
-- package: github.com/tendermint/go-config
-- package: github.com/tendermint/go-crypto
-- package: github.com/tendermint/go-db
-- package: github.com/tendermint/go-events
-- package: github.com/tendermint/go-logger
-- package: github.com/tendermint/go-merkle
-- package: github.com/tendermint/go-p2p
 - package: github.com/stretchr/testify
-- package: github.com/tendermint/go-rpc
-- package: github.com/tendermint/go-wire
```

Comment:
the use of go-wire in the state of edb is logically disconnected from the use of go-wire inside of tendermint, but this should not be a pressing issue today